pytorch_to_returnn/torch/nn/modules/variable.py

```diff
@@ -17,16 +17,25 @@ def __init__(self, param: Parameter):
     super(Variable, self).__init__()
     assert isinstance(param, Parameter)
     self.param = param
+    self.parent_mod = None
 
   def create_returnn_layer_dict(self, *inputs):  # ignore inputs
     return {"class": "variable", "add_batch_axis": False, "shape": self.param.shape}
 
   def make_output_tensor_from_returnn(self, inputs_flat: List[Tensor], layer: LayerBase) -> Tensor:
+    naming = Naming.get_instance()
+    values = self.param.detach().numpy()
```

Comment:
Better set `values = None` here, and then below sth like `if values is None: values = self.param.detach().numpy()`.